bin/postman-test/collections/support-logging.postman_collection.json

```diff
@@ -706,18 +705,18 @@
 									"     pm.expect(result).to.be.true;",
 									"});",
 									"",
-									"pm.test(\"Collection's size should not exceed the configured ReadMaxLimit.\", function () {",
+									"pm.test(\"Collection's size should not exceed the configured ReadMaxLimit. Please check the MaxResultCount value on service configuration. If the value is greater than 100, this case will fail. \", function () {",
```

Comment:
Please check the MaxResultCount value on service configuration. If the value is greater than 100, this case will fail.  You could run or refer to bin/updateMaxResultCount.sh before this test.

Reply:
Updated.